src/components/masters/program/ProgramPage.jsx

```diff
@@ -81,11 +81,16 @@ class ProgramPage extends Component {
       programDocuments,
       programBranding,
       externalProgramWebsite,
+      programHostname,
+      programSlug,
     } = pageContext;
 
     return (
       <IntlProvider locale="en">
-        <Layout>
+        <Layout
+          programHostname={programHostname}
+          programSlug={programSlug}
+        >
```

Comment:
If a use is on a program page they do not have access too, this will pull the slug for that page into the layout, making the redirect move to a non-accessible program. I don't know if this is a problem right now? (since its just redirecting them to a page they are already on anyway).

Reply:
I will think about this, I am not sure what the right answer will be as we do not pull the program information from designer again.